src/components/BoxBackground/BoxBackground.test.js

```diff
@@ -1,70 +1,99 @@
 
-  afterEach(() => {
-    wrapper.unmount()
-  })
+    const boxBackgroundNode = getByText(children)
 
-  it('is mounted', () => {
-    expect(wrapper.exists()).toBe(true)
+    expect(boxBackgroundNode).toBeVisible()
   })
 
-  it('has `boxBackground` class', () => {
-    expect(wrapper.children().hasClass('boxBackground')).toBe(true)
+  it(`should contains boxBackground class `, () => {
+    const { getByText } = render(<BoxBackground {...initialProps} />)
+
+    const boxBackgroundNode = getByText(children)
+
+    expect(boxBackgroundNode).toHaveClass('boxBackground')
   })
 
   it('has proper class name on success variant', () => {
-    wrapper.setProps({
+    const newProps = {
+      ...initialProps,
       variant: 'success'
-    })
+    }
+
+    const { variant } = newProps
+
+    const { getByText } = render(<BoxBackground {...newProps} />)
 
-    expect(wrapper.children().hasClass('boxBackground--success')).toBe(true)
+    const boxBackgroundNode = getByText(children)
+
+    expect(boxBackgroundNode).toHaveClass(`boxBackground--${variant}`)
   })
 
   it('has proper class name on warning variant', () => {
-    wrapper.setProps({
+    const newProps = {
+      ...initialProps,
       variant: 'warning'
-    })
+    }
+
+    const { variant } = newProps
 
-    expect(wrapper.children().hasClass('boxBackground--warning')).toBe(true)
+    const { getByText } = render(<BoxBackground {...newProps} />)
+
+    const boxBackgroundNode = getByText(children)
+
+    expect(boxBackgroundNode).toHaveClass(`boxBackground--${variant}`)
   })
 
   it('has proper class name on alert variant', () => {
-    wrapper.setProps({
+    const newProps = {
+      ...initialProps,
       variant: 'alert'
-    })
+    }
+
+    const { variant } = newProps
+
+    const { getByText } = render(<BoxBackground {...newProps} />)
+
+    const boxBackgroundNode = getByText(children)
 
-    expect(wrapper.children().hasClass('boxBackground--alert')).toBe(true)
+    expect(boxBackgroundNode).toHaveClass(`boxBackground--${variant}`)
   })
 
   it('has proper class name on progress variant', () => {
-    wrapper.setProps({
+    const newProps = {
+      ...initialProps,
       variant: 'progress'
-    })
+    }
 
-    expect(wrapper.children().hasClass('boxBackground--progress')).toBe(true)
+    const { variant } = newProps
+
+    const { getByText } = render(<BoxBackground {...newProps} />)
+
+    const boxBackgroundNode = getByText(children)
+
+    expect(boxBackgroundNode).toHaveClass(`boxBackground--${variant}`)
   })
 
   it('has proper class name on info variant', () => {
-    wrapper.setProps({
-      variant: 'info'
-    })
+    const { variant } = initialProps
+
+    const { getByText } = render(<BoxBackground {...initialProps} />)
+
+    const boxBackgroundNode = getByText(children)
 
-    expect(wrapper.children().hasClass('boxBackground--info')).toBe(true)
+    expect(boxBackgroundNode).toHaveClass(`boxBackground--${variant}`)
```

Comment:
tu to samo, wcale nie sprawdzasz, czy to ma klasę 'info' tylko czy tworzy klasę `boxBackground--${variant}`. Z takim podejściem można te dwa testy połączyć w jeden, łączenie stringów nie zacznie  źle działać w zależonosci od tego czy jest 'alert' czy 'info'